src/main/java/com/criticalsoftware/Annoucement.java

```diff
@@ -0,0 +1,64 @@
+package com.criticalsoftware;
+
+import io.quarkus.mongodb.panache.PanacheMongoEntity;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public class Annoucement extends PanacheMongoEntity {
+
+    private Product product;
+    private LocalDateTime date;
+    private User userDonor;
+    private User userDonee;
+
+    public Annoucement() {
+
+    }
+
+
+    public Annoucement(Product product, LocalDateTime date, User userDonor, User userDonee) {
+        this.product = product;
```

Comment:
no need for date to passed into the method.
You can just write: this.date = LocalDateTime.now()

It just saves the date at that instant.